src/main/java/emu/grasscutter/command/commands/CommandPermissionCommand.java

```diff
@@ -0,0 +1,57 @@
+package emu.grasscutter.command.commands;
```

Comment:
This command should probably be dropped, and just make `help <command>` show the permission information as mentioned in the other comment.

Reply:
I removed that command.

Reply:
Added permission displaying for the single command.